### PR TITLE
add a file with instructions for contributing

### PR DESCRIPTION
**Pull request by npapernot**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Contributing to CleverHans
+
+First off, thank you for considering contributing to CleverHans.
+Following these guidelines helps to communicate that you respect
+the time of the researchers and developers managing and developing this open
+source project. In return, they should reciprocate that respect in
+addressing your issue, assessing changes, and helping you finalize
+your pull requests.
+
+Improving documentation, bug triaging, or writing tutorials are all
```
Comment (goodfeli) [inline CONTRIBUTING.md]: Why not new features?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Contributing to CleverHans
+
```
Comment (goodfeli) [inline CONTRIBUTING.md]: Maybe we should have a message saying that if you publish a new attack or defense you're highly encouraged to add it to CleverHans so people can evaluate it fairly?

**Comment by goodfeli** [comment]
@npapernot : this is ready once you fix the two comments above

**Comment by npapernot** [comment]
OK, taking care of it now and will merge

**Pull request merged**